### DISCUSSION TITLE
v2.1.6

### DISCUSSION THREAD

**Discussion by rkitover** [visualboyadvance-m/visualboyadvance-m#1148] [General]
* 7561ca97 - Fix Discord streaming for OpenGL [danialhorton]
* e26f8073 - [bios] Fix lz77 and add BIOS_SndDriverVsyncOn [steelskin]
* d1f65000 - Refactor accelerator / global shortcuts handling [steelskin]
* fda429fc - [GBA] Do not draw BG tiles from outside of BG VRAM [steelskin]
* 5aa3ea92 - Properly initialize wxAcceleratorTable [steelskin]
* 0e29be87 - Support multiple RAM sizes for MBC7 [steelskin]
* 75b79d91 - Change MBC7 reported RAM size to 512 bytes [steelskin]
* 36e88faf - Fix dsound looping when moving/resizing window [rkitover]
* 019fcda9 - Sanitize and improve the GB header parsing (#1109) [Steelskin]
* 7005b928 - Sanitize allocations in the Game Boy emulator (#1105) [Steelskin]
* 3cef52b7 - Fix pause/next-frame [rkitover]
* d3f8cc17 - Option to suspend the X11 screensaver when running [2306079+multiSnow]
* df819598 - builder: update macOS build to 10.10, Wx 3.2.2.1 [rkitover]
* 1ab46f70 - Update SIOCNT when the link driver is disconnected [steelskin]
* 1d7e8ae4 - build: fix build with new OpenAL [rkitover]
* a9284b3d - workaround for Stuart Little 2 crashing when set to none [danialhorton]
* 166299cd - Implements Read/Writes for HUC3 [danialhorton]
* 640ce453 - Change SOUND_CLOCK_TICKS to 280896 cycles - ~1074 samples per frame [reallibretroretroarch]
* f1d3f631 - Fix automatic dynamic frame skipping [steelskin]
* d756f671 - translations: fix some source strings [stanley.udr.kid]
* 91873254 - Add INI file versioning [steelskin]
* 7d9cb2b5 - Default GBA LCD filter to off. [rkitover]
* be1e5ca1 - Check for wx < 3.2.2 for EGL move fix backport [rkitover]
* 1fb18b32 - Rename appdata to metainfo [polynomial-c]
* ca7ae336 - Fix regression in bios, port DMA fix from libretro [danialhorton]
* b34448c9 - Fix record/play dialogs for native recordings [danialhorton]
* 6307348c - build: support MSVC arm64 cross build on x86 [rkitover]
* 87bdc93e - Revert "translations: use %% instead of percent" [rkitover]
* ab4ae96e - Clarify/set condition: wayland && !wayland_egl [rkitover]
* 8e4acfc9 - Fix build w/wxUSE_GLCANVAS_EGL=0, on Fedora [rkitover]
* 3c41b68d - Fix the Windows ARM64 build [steelskin]
* 22578d02 - Increase MAX_CART_SIZE from 32MiB to 128MiB [rkitover]

<hr /><em>This discussion was created from the release <a href='https://github.com/visualboyadvance-m/visualboyadvance-m/releases/tag/v2.1.6'>v2.1.6</a>.</em>

**Comment by dftf-stu** [comment]
I'm not able to download the x86_64 ZIP file currently on Windows 10: it gets detected as a virus called "Trojan:Script/Wacatac.B!ml" and immediately removed. 😥 VirusTotal [says](https://www.virustotal.com/gui/file/bcd70864de66d324484f3aa5ce41540e3389be99e06fabed1201a8009b25c8fa) no-other AV currently flags it, so I've submitted a false-detection report to Microsoft [here](https://www.microsoft.com/en-us/wdsi/submission/8b082606-36c3-47ab-bafa-e6342c61a3fe).

**Reply by turkish-kamerad** [reply]
try turning your anti virus system off and theres tutorials on how to solve this if it doesnt work

**Comment by turkish-kamerad** [comment]
I downloaded all of them know how do I update it should I just put all of them in to the emulator?

**Comment by raisaab198** [comment]
Firstly, thank you very much for keeping the development for this nice emulator on. My issue is about the "real time clock" in Pokemon Fire Red v1.1 (USA, EU) rom. I have tried to install vba-m version 2.0.2, version 2.1.6 and nightly as well but no luck. I tried to change the emulator settings to turn on real time clock and also followed those solutions told by you to save as flash 128. I remember that I played this game around 8 years ago and this was working at that time. Maybe there is a compatibility issue with windows 10 or maybe any permission issue or maybe the rom file issue, I have no idea. Please let me know if there is any solution because those pokemons cannot appear in daytime like hoothoot. So for that we need real time clock mathed with pc or maybe there can be an add on to change time within vba? Thank You very much

**Comment by Squall-Leonhart** [comment]
Fire red turns on the RTC by default, the time of day is inheritted from the system clock

**Reply by raisaab198** [reply]
Thank you very much. I have no idea how can I check it because when I open menu with my player id name, it shows me different time than in my computer. However, I was wrong about night time in firered because I was confused it with heartgold. In HeartGold, you can catch pokemons at certain time. I apologize for my mistake. Thanks again and have a nice weekend

**Reply by Squall-Leonhart** [reply]
oh, yeah lol, the Fire Red and Leaf Green games don't have RTC, they are pure adaptions of the original gb games.
